### ISSUE TITLE
getTransactionsToApprove missing input validation

### ISSUE THREAD

**Issue by pinpong** [iotaledger/iri#135]
````
[XNIO-1 task-4] ERROR com.iota.iri.service.API - API Exception:
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Double
        at com.iota.iri.service.API.process(API.java:284) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.processRequest(API.java:147) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API.access$000(API.java:81) [iri-1.1.3.10.jar:na]
        at com.iota.iri.service.API$1.handleRequest(API.java:135) [iri-1.1.3.10.jar:na]
        at io.undertow.server.Connectors.executeRootHandler(Connectors.java:208) [iri-1.1.3.10.jar:na]
        at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:809) [iri-1.1.3.10.jar:na]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_131]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_131]
        at java.lang.Thread.run(Thread.java:748) [na:1.8.0_131]
> [XNIO-1 task-6] ERROR com.iota.iri.service.API - API Exception:
> java.lang.ClassCastException: null

**Comment by Synergist** [comment]
Shouldn't this be closed by alon-e@7ed8fe82?

**Comment by pinpong** [comment]
This issue is already fixed.
